packages/oui-table/_mixins.less

```diff
@@ -156,6 +171,111 @@
     }
   }
 
+  .table-responsive(
+    @selector
+  ) {
+    .tablet({
+      table-layout: auto;
+
+      &__headers {
+        display: block;
+        position: absolute;
+        left: -9999em;
+      }
+
+      &__body {
+        .@{selector}__row {
+          display: block;
+          float: left;
+          margin-right: 1%;
+          margin-bottom: 10px;
+          width: 49.5%;
+
+          &:nth-child(odd) {
+            clear: left;
+          }
+
+          &:nth-child(even) {
+            margin-right: 0px;
+          }
+
+          &:hover {
+            background:none;
+          }
+        }
+      }
+
+      &__cell {
+        display: block;
+        border: none;
+
+        &[data-title]:before {
+          color: #676767;
+          content: attr(data-title);
+          display: block;
+          font-size: rem-calc(14);
+        }
+      }
+
+      &__cell, &__header {
+        border: none;
+        border-left: 1px solid #DFE0E5;
+        border-right: 1px solid #DFE0E5;
+
+        &:first-child {
+          border-top: 1px solid #DFE0E5;
+        }
+
+        &:last-child {
+          border-bottom: 1px solid #DFE0E5;
+        }
+      }
+    });
+
+    .phone({
+      &__body {
+        .@{selector}__row {
+          margin-bottom: 0;
+          width:100%;
+
+          .@{selector}__cell {
+            .@{selector}__expand-button {
+              display: block;
+            }
+
+            &:last-child {
+              border-bottom: none;
+            }
+          }
+
+          &_closed {
+            .@{selector}__cell {
+              display: none;
+
+              &[data-title]:before {
```

Comment:
We need to be more cautious when class are used inside mixin parameter, `stylelint` just stop its linting and pass to the next parameter.